src/Okta.Sdk/Internal/DefaultSerializer.cs

```diff
@@ -23,14 +23,22 @@ public sealed class DefaultSerializer : ISerializer
         /// <summary>
         /// Initializes a new instance of the <see cref="DefaultSerializer"/> class.
         /// </summary>
-        public DefaultSerializer()
+        /// <param name="settings">settings to customize how JSON is serialized.</param>
+        public DefaultSerializer(JsonSerializerSettings settings = null)
         {
-            _serializer = new JsonSerializer
+            if (settings != null)
             {
-                NullValueHandling = NullValueHandling.Ignore,
-                DefaultValueHandling = DefaultValueHandling.Ignore,
-                ContractResolver = new DefaultContractResolver(),
-            };
+                _serializer = JsonSerializer.Create(settings);
+            }
+            else
+            {
+                _serializer = new JsonSerializer
+                {
+                    NullValueHandling = NullValueHandling.Ignore,
+                    DefaultValueHandling = DefaultValueHandling.Ignore,
+                    ContractResolver = new DefaultContractResolver(),
```

Comment:
This ContractResolver. We normally use a DefaultContractResolver but if a dev passes in a custom settings then it will use whatever is in there, not sure if it has the same default of if it defaults to null or what, or frankly what it even does in the first place. I doubt it’s an issue, just wanted to see if you thought it’s worth looking into.